src/registrar/templates/admin/base_site.html

```diff
@@ -13,5 +13,20 @@ <h1 id="site-name"><a href="{% url 'admin:index' %}">.gov admin</a></h1>
   {% include "admin/color_theme_toggle.html" %}
 {% endif %}
 {% endblock %}
-
+{% block userlinks %}
```

Comment:
**comment (non-blocking)**: It looks like this was probably copied out of the default Django admin templates so that it could be modified. If so, it might be nice to include a link back to the source in case Django modifies the structure of the base template in the future and we need to change this block to match.